src/js/components/Anchor/doc.js

```diff
@@ -2,6 +2,52 @@ import { describe, PropTypes } from 'react-desc';
 
 import { genericProps, getAvailableAtBadge } from '../../utils';
 
+export const themeDoc = {
+  'global.focus.border.color': {
+    description: 'The color around the Anchor when in focus.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: '#FD6FFF',
+  },
+  'anchor.color': {
+    description: 'Used to apply a default color to the icon and label.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: "{ light: '#1D67E3', dark: '#6194EB' }",
+  },
+  'anchor.fontWeight': {
+    description: 'The font weight to be applied to the anchor text.',
+    type: 'number',
+    defaultValue: 600,
+  },
+  'anchor.textDecoration': {
+    description: 'The text decoration to be applied to the anchor.',
```

Comment:
Can we include a reference to the CSS attribute when there is a clear correlation? That will help users know what values are permitted.

Reply:
agreed.